lib/cancan/model_adapters/active_record_5_adapter.rb

```diff
@@ -0,0 +1,58 @@
+module CanCan
+  module ModelAdapters
+    class ActiveRecord5Adapter < AbstractAdapter
```

Comment:
Are there really any differences between the active record 4 adapter and this one? Besides removing version checks?

If we have to keep this as a seperate file, it needs its own tests. If possible though (and if they are as similar as I presume) I would like to make the active record 4 adapter work for both major versions. Unless there is a reason to presume otherwise. I may be missing something here so feel free to enlighten me!